### DISCUSSION TITLE
BottomSheetBackdrop will not appear if there is only one snap point.

### DISCUSSION THREAD

**Discussion by RamsayRomero** [gorhom/react-native-bottom-sheet#1566] [Q&A] [answered]
If I only set one snap point, there's nothing I can do to get the BottomSheetBackdrop to appear. Since only the 0th index exists, the bottom sheet should appear if I set `appearsOnIndex` to 0. It doesn't show, and will actually never show no matter how many snap points I have. I can't think of a reason why having the backdrop appear at the first index should be impossible. To me this feels like a bug. If it is impossible, it should be noted in the docs that 0 is not a valid value for this prop. Am I missing something? How can I get the backdrop to appear at the first/only snap point?

**Comment by MichaelTao** [comment]
@RamsayRomero Did you ever solve this? My backdrop doesn't appear either... using BottomSheetModal

**Reply by RamsayRomero** [reply, marked as answer]
Yes, you just need to set the `disappearsOnIndex` prop of the `BottomSheetBackdrop` component to -1. The default is 0 so it will never appear on the 0th index. I think this is an unintuitive default and the library should consider changing it.
